module/behaviour/skills/Getup/README.md

```diff
@@ -11,15 +11,19 @@ Include this module to allow the robot to get up after it has fallen over.
 
 ## Consumes
 
-* `message::Sensors` containing the orientation matrix and accelerometer readings
-* `message::AllServoWaypointsComplete` indicates that a script has finished
+* `message::motion::ExecuteGetup` tells this module to call getup script
```

Comment:
Seems fine, though why not use hyphen for lists, to match formatting in NUbook 😛 

```suggestion
- `message::motion::ExecuteGetup` tells this module to call getup script
```